VAMobile/documentation/docs/Engineering/FrontEnd/DevelopmentBestPractices.md

```diff
@@ -0,0 +1,130 @@
+# Development Best Practices
+
+Our team follows a set of best practices around tickets, branches, and PRs to ensure a smooth development process and high quality code. Here we'll describe the key points to keep in mind.
+
+## Tickets
+
+We use tickets to track all our engineering work. They precisely specify the work to be performed, allow stakeholders to track progress, and guide QA in assessing the resulting work.
+
+### Creating tickets
+
+Every ticket must contain at least the following information:
+
+- A detailed description of the work
+- Clearly specified Acceptance Criteria
+- Screenshots or links to mockups where applicable
+- All relevant labels ("front-end" for FE work, "Health" for Health and Benefits work, "Global" for global work, etc)
+- An epic relating to the ticket's larger context
+
+If possible, choose the relevant template for your ticket. For example, if you're creating a bug ticket, choose the Bug Report Template and include severity in the title, labels, and body.
+
+### Working on tickets
+
+All engineering work requires a ticket.
+
+Every ticket must satisfy the requirements in the "Creating tickets" section above. If information is missing, add it or contact the relevant stakeholders.
+
+Before an engineer can begin work on a ticket, it must contain an estimate. Bugs get an initial estimate of 1 to cover a short investigation. Other tickets are left to the engineers' discretion.
+
+Once everything is in place, complete the following tasks:
+
+- Assign yourself
+- Assign the appropriate QA engineer if not already assigned
+- Move the ticket to In Progress
+- Create a branch as described below in the Branches section
+
+Then you can begin work. Keep stakeholders up to date as you work by commenting on the ticket with any important details or questions. This will also preserve information so everyone can reference it in the future.
+
+On completion of your work, double check the following:
+
+- All items in Acceptance Criteria are implemented
+- Confirm screen screen reader functionality on both iOS and Android for accessibility
+- Unit and E2E tests are updated or extended to cover your changes
+- Relevant documentation is updated to reflect the changes in your PR
+
+Then create a Pull Request. See the Pull Requests section below for more information.
+
+## Branches
+
+Feature branches are temporary branches where we develop and test new features. We implement every code change in a feature branch before merging it into the default branch.
+
+:::info
+For features that must be rolled out gradually, or where precise feature activation timing is required, use a [remote config flag](https://firebase.google.com/docs/remote-config/).
+:::
+
+Our default branch is `develop`. Create your feature branch from there, named as follows:
+
+```
+type/ticketnumber-description-of-work
```

Comment:
Intentional to no longer include name of who made the branch?

Separately, existing practice has been generally (but not consistently) using PascalCase for the `description-of-work` section which I think makes more sense because we can use `-`'s to add additional descriptors (e.g. `bug/####-Android-BugDetails`).

Reply:
Yeah, see Tim's comment above about branch naming.

I updated to PascalCase since I agree, that's what we generally do as a team. There can be some weirdness around case sensitivity with GitHub, but I haven't noticed it being a problem for us